.travis.yml

```diff
@@ -2,9 +2,10 @@ sudo: false
 language: python
 python:
   - "2.7"
-  - "3.3"
-  - "3.4"
   - "3.5"
+  - "3.6"
```

Comment:
Updated according to https://devguide.python.org/#branchstatus

Thanks @kosmos342 for suggestion and examples from https://github.com/yola/demands/pull/76/files

Reply:
Also, https://www.python.org/downloads/release/python-3410/ says that "Python 3.4 has reached end-of-life".